src/VariableInput/VariableInput.uni.driver.js

```diff
@@ -6,6 +6,8 @@ export const getContent = base => base.$('.public-DraftEditor-content');
 export default (base, body) => {
   return {
     ...baseUniDriverFactory(base, body),
+    isDisabled: async () =>
+      Boolean(await getContent(base).attr('contenteditable')),
```

Comment:
I think it should be:
```suggestion
      !Boolean(await getContent(base).attr('contenteditable')),
```

Because when you've `contenteditable` it means the content isn't readonly, right?

Reply:
Actually, I took this example from RichTextInputArea, we always got `contenteditable` attribute, when disabled it is set to `false`,  but it is not clear.
Will push a fix, because: `Boolean('false') === true`